infrastructure/kctf/scripts/cluster/start.sh

```diff
@@ -18,6 +18,35 @@ gcloud compute routers nats create kctf-${CLUSTER_NAME}-nat-config --router-regi
 
 get_cluster_creds
 
+# GCSFUSE
+
+SUFFIX=$(echo "${PROJECT}-${CLUSTER_NAME}-${ZONE}" | sha1sum)
+BUCKET_NAME="kctf-gcsfuse-${SUFFIX:0:16}"
+GSA_NAME="${BUCKET_NAME}"
+GSA_EMAIL=$(gcloud iam service-accounts list --filter "name:${GSA_NAME}" --format 'get(email)' || true)
+if [ -z "${GSA_EMAIL}" ]; then
+  gcloud iam service-accounts create "${GSA_NAME}" --description "kCTF GCSFUSE service account ${CLUSTER_NAME} ${ZONE}" --display-name "kCTF GCSFUSE ${CLUSTER_NAME} ${ZONE}"
+  GSA_EMAIL=$(gcloud iam service-accounts list --filter "name:${GSA_NAME}" --format 'get(email)')
+fi
+if ! gsutil du "gs://${BUCKET_NAME}/"; then
+  gsutil mb -l eu "gs://${BUCKET_NAME}/"
+  gsutil uniformbucketlevelaccess set on "gs://${BUCKET_NAME}/"
+fi
+gcloud projects add-iam-policy-binding "${PROJECT}" --member "serviceAccount:${GSA_EMAIL}" --role roles/storage.objectAdmin
+
+KEY_PATH=$(mktemp -d)/key.json
+
+gcloud iam service-accounts keys create "${KEY_PATH}" --iam-account "${GSA_EMAIL}"
```

Comment:
can you check for an existing key to use?

Reply:
do you mean check if it already exists in the cluster? or in gcloud?

if it already exists in gcloud, we might still want to recreate it (say, if you reset the cluster), and if it already exists in the cluster, we also might want to recreate it (say, if someone deleted it from pantheon)

Reply:
I was mainly thinking that we should clean up so that we don't have 20 keys after a few cluster start/stop ops.
I guess we can just delete the key on cluster stop since it will be useless anyway? But yeah, as long as the call doesn't file when one already exists it's not so important.

Reply:
ah ok, we can delete the GSA when we stop the cluster, it's fast to create anyway and more polite